python/pyspark/ml/tuning.py

```diff
@@ -112,15 +198,24 @@ class CrossValidator(Estimator, HasSeed):
     >>> cvModel = cv.fit(dataset)
     >>> evaluator.evaluate(cvModel.transform(dataset))
     0.8333...
+    >>> cvPath = temp_path + "/cv"
+    >>> cv.save(cvPath)
+    >>> loadedCV = CrossValidator.load(cvPath)
+    >>> loadedCV.getEstimator().uid == cv.getEstimator().uid
```

Comment:
Ah.. Sorry for forgetting there are unit tests for CrossValidator and TrainValidateSplit in tests.py.